iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -446,25 +462,15 @@ TEST_F(PortManager_test, AcquiringMaximumNumberOfEventVariablesWorks)
 {
     std::string process = "BuddyHolly";
 
-    for (unsigned int i = 0U; i < iox::MAX_NUMBER_OF_EVENT_VARIABLES; i++)
-    {
-        iox::ProcessName_t newProcessName(iox::cxx::TruncateToCapacity, process + std::to_string(i));
-        auto eventVariableDataResult = m_portManager->acquireEventVariableData(newProcessName);
-        EXPECT_THAT(eventVariableDataResult.has_error(), Eq(false));
-    }
+    acquireMaxNumberOfEventVariables(process);
```

Comment:
It looks a bit lonely without an `EXPECT`. What do you think about `acquireMaxNumberOfEventVariables` returning the number of successful acquired event variables and do a check on that.
Not strictly necessary since the method already checks for valid ports, but could prevent some `what the heck` moments in the future when just reading the test.